backend/dataall/core/permissions/db/permission/permission_repositories.py

```diff
@@ -1,10 +1,9 @@
 import logging
 
-from dataall.core.permissions.db.permission_models import PermissionType
+from dataall.core.permissions.api.enums import PermissionType
 from dataall.base.db import exceptions
-from dataall.core.permissions import permissions
-from dataall.core.permissions.db import permission_models as models
-
+from dataall.core.permissions.constants import permissions
+from dataall.core.permissions.db.permission import permission_models as models
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Similar comments than the ones in Resource_Policy: 
1. we should rename this to PermissionsRepository
2. all the business logic should be placed in the services layer, so maybe we need to create a `permissions_service.py` that contains functions such as `get_permission_by_name`...